src/components/InputField.js

```diff
@@ -0,0 +1,38 @@
+import {Component} from "react";
+import './InputField.css';
+
+class InputField extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            text: ''
```

Comment:
Is it necessary to store input value in the state? If in **handleKeyDown** argument **e** has a value of the input then it's better to make this component not controlled (refer to controlled and not controlled components in react docs).